params/config.go

```diff
@@ -1041,7 +1049,7 @@ func (c *ChainConfig) Rules(num *big.Int, isMerge bool, timestamp uint64) Rules
 		chainID = new(big.Int)
 	}
 	// disallow setting Merge out of order
-	isMerge = isMerge && c.IsLondon(num)
+	isMerge = c.IsLondon(num)
```

Comment:
https://github.com/ethereum/go-ethereum/blob/v1.13.15/params/config.go#L914
https://github.com/bnb-chain/bsc/blob/v1.4.15/params/config.go#L1395
```suggestion
	isMerge = isMerge && c.IsLondon(num)
```